src/components/application_manager/include/application_manager/commands/mobile/set_app_icon_request.h

```diff
@@ -89,7 +89,8 @@ class SetAppIconRequest : public CommandRequestImpl {
    * @param icons_amount Amount of icons to be deleted
    */
   void RemoveOldestIcons(const std::string& storage,
-                         const uint32_t icons_amount) const;
+                         const uint32_t icons_amount,
+                         bool& is_correct_removed) const;
```

Comment:
@VVeremjova Not mandatory, but it will be good to change the name of the third parameter from is_correct_removed to is_correct_removed_out, because it is an out parameter.